office/settings.py

```diff
@@ -85,10 +85,10 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'HOST': os.getenv('POSTGRES_HOST', 'database'),
+        'HOST': os.getenv('POSTGRES_HOST', 'localhost'),
```

Comment:
If you run the project via the Docker container, the value `database` will be suitable for you.
I wouldn't change the values. It doesn't solve any problem.

Reply:
It was by mistake